src/components/QuestionList.js

```diff
@@ -30,18 +30,28 @@ class QuestionList extends React.Component {
       deleteId: null,
       cancelId: null,
     }
+
+    if (props.course && props.course.questionFeedback) {
+      this.state.shouldShowFeedbackModal = true
```

Comment:
This shouldn't be stored in state - this should be computed from `props` whenever it's needed. If left as-is, this won't properly update if our props change.